src/index.ts

```diff
@@ -2,6 +2,8 @@ export * from './client.js';
 export * from './constants.js';
 export * from './deck/index.js';
 export * from './fetch-map/index.js';
+export {LayerType} from './fetch-map/layer-map.js';
+export {LayerDescriptor} from './fetch-map/parse-map.js';
```

Comment:
Could we export these from `./fetch-map/index.ts` if they're intended to be public?

Reply:
Done